src/com/stanimiriliev/chaptertwo/auto/Automobiles.java

```diff
@@ -0,0 +1,27 @@
+package com.stanimiriliev.chaptertwo.auto;
+
+import java.text.DateFormat;
+import java.util.Date;
+
+/**
+ * Represent the common properties of the cars of several makes
+ * @author Stanimir Iliev
+ * @since 22.07.2017
+ */
+public class Automobiles {
```

Comment:
I made the class Automobile now abstract and use commit --amend and then push and I accidently open new PR